vulnerabilities/improvers/vulnerability_kev.py

```diff
@@ -28,39 +28,40 @@ def get_inferences(self, advisory_data) -> Iterable[Inference]:
         Fetch Kev data, iterate over it to find the vulnerability with the specified alias, and create or update
```

Comment:
Please us the `aboutcode.pipeline` here too. Let me know if you need any help with it.

Reply:
Done